app/src/main/res/layout/list_steps.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        android:layout_width="match_parent"
+        android:orientation="horizontal"
+        android:layout_height="wrap_content">
+
+    <TextView
+            android:id="@+id/txt_short_description"
+            android:padding="@dimen/standard_padding"
+            android:layout_width="@dimen/dimen_zero"
```

Comment:
@KhinThiriSoe 

don't use `dimen_zero` here. You can use variable names that literally represent actual values only for ConstraintLayout. 

This needs to have a proper English name

Reply:
<img width="643" alt="screen shot 2019-03-08 at 4 34 32 pm" src="https://user-images.githubusercontent.com/10254643/54020513-860c7380-41c0-11e9-8cbf-83c4da522257.png">

Reply:
@KhinThiriSoe I see because you have used weight. Ok, will resolve this one